qa/integration/src/test/resources/features/jobEngine/JobEngineServiceRestartOfflineDeviceI9n.feature

```diff
@@ -9,10 +9,9 @@
 # Contributors:
 #     Eurotech - initial API and implementation
 ###############################################################################
-@integration
-@jobs
 @jobEngineService
 @jobEngineRestartOfflineDevice
+@integration
```

Comment:
Same comment as above; if you wanted to put "integration" tags in the end it's ok, otherwise put it back.